# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/NotificationUtils.h
# Commit: f9f3e107f36c
# Full Hash: f9f3e107f36c8cce438711ecef1b1a9574ae8f93
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2025-03-22 09:02:07
# Regressor Bug: 1948339
# File Overlap Count: 2
# Description:
#   Bug 1948339 - Part 2: Add NotificationObserver r=asuth
#   
#   This also makes page-bound notifications to not be stored via NotificationStorage.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D240642
# ==============================================================================

diff -r b84ed8a9da4c -r f9f3e107f36c dom/notification/NotificationUtils.h
--- a/dom/notification/NotificationUtils.h	Fri Mar 21 09:52:31 2025 +0000
+++ b/dom/notification/NotificationUtils.h	Fri Mar 21 09:52:31 2025 +0000
@@ -80,8 +80,7 @@
   // Either on global teardown or freeze
   InactiveGlobal,
 };
-void UnregisterNotification(nsIPrincipal* aPrincipal, const nsString& aId,
-                            CloseMode aCloseMode);
+void UnregisterNotification(nsIPrincipal* aPrincipal, const nsString& aId);
 
 nsresult RemovePermission(nsIPrincipal* aPrincipal);
 nsresult OpenSettings(nsIPrincipal* aPrincipal);